docs/modules/ROOT/pages/troubleshooting/faqs.adoc

```diff
@@ -34,3 +34,57 @@ type Cookie {
     # f: String # If you don't want _emptyInput, uncomment this line.
 }
 ```
+
+== Relationship nullability isn't being enforced in my graph
+Currently, Neo4j GraphQL enforces on creating and updating that a movie has a director: 
```

Comment:
An example has not been set here, so `enforce ... that a movie has a director` may not be easy to understand, Maybe use a more generic description and add this referencing the example below?

Reply:
Sure!